src/fixate/config/__init__.py

```diff
@@ -51,6 +54,7 @@
         "test-script-name={test_script_name}",
         "report-format={REPORT_FORMAT_VERSION}",
         "index_string={index}",
+        "computername={COMPUTERNAME}",
```

Comment:
Just noting in case you haven't followed it through before but this default config is overwritten when called through amptest. So will need to be updated in the src there too,